.github/workflows/publish.yml

```
@@ -0,0 +1,81 @@
+    needs: build # Only upload to release if build succeeds
+    permissions:
+      contents: write  # Required to upload assets to the release
+    steps:
+      - name: Download built package
+        uses: actions/download-artifact@v4
+        with:
+          name: package-distributions
+          path: dist/
+
+      - name: Upload to GitHub Release
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.4)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)